File: 백준/Bronze/15873. 공백 없는 A＋B/공백 없는 A＋B.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int b = 0, a = sc.nextInt();
        
        if (a == 1010){         //1010
            System.out.println(20);
        }
        else if (a % 10 == 0){  //a10
            a /= 100;
            System.out.println(a+10);
        }
        else{
            b += a % 10;
            a /= 10;
            if (a == 10){       //10b
                System.out.println(10+b);
            }
            else{               //ab
                System.out.println(a+b);
            }
        }
    }
}

File: 백준/Bronze/15873. 공백 없는 A＋B/공백 없는 A＋B.py
n = input()
if len(n) == 4:           #1010
    print(20)
elif len(n) == 2:         #ab
    print(int(n[0]) + int(n[1]))
else:
    if int(n[-1]) == 0:   #a10
        print(int(n[0]) + 10)
    else:                 #10b
        print(int(n[-1]) + 10)